### PR TITLE
Update creating_a_webhook.md

### PR DESCRIPTION
**Pull request by SimeonBrazeSA**
### Summary

Updating docs to make this clearer on the back of this PQ: https://servicedesk.braze.com/plugins/servlet/desk/portal/12/PQ-4161

### Related PRs, issues, or features (optional)

- N/A

### Feature release date (optional)

- N/A

### Contributor checklist

- [x] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/webhooks/creating_a_webhook.md [hunk 1]
```diff
@@ -200,7 +200,7 @@ Like other campaigns, Braze tracks the delivery of your webhook campaigns and co
 | 5xx (server error)   | No | Yes |
 {: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}
 
-Braze will retry webhook sends for up to 24 hours. When retrying, Braze will make five attempts using exponential backoff for a period of approximately 30 minutes before aborting the individual webhook call.
+Braze will retry webhook sends for up to 24 hours before aborting the individual webhook call. When retrying after 5xx errors, Braze will make five attempts using exponential backoff for a period of approximately 30 minutes before aborting the individual webhook call.
```
Comment (internetisaiah) [inline _docs/_user_guide/message_building_by_channel/webhooks/creating_a_webhook.md:203]:
```suggestion
Braze will retry webhook sends for up to 24 hours before aborting the individual webhook call. When retrying after any 500-level error, Braze will make 5 attempts using exponential backoff for a period of approximately 30 minutes before aborting the individual webhook call.
```

File: _docs/_user_guide/message_building_by_channel/webhooks/creating_a_webhook.md [hunk 2]
```diff
@@ -200,7 +200,7 @@ Like other campaigns, Braze tracks the delivery of your webhook campaigns and co
 | 5xx (server error)   | No | Yes |
 {: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}
 
-Braze will retry webhook sends for up to 24 hours. When retrying, Braze will make five attempts using exponential backoff for a period of approximately 30 minutes before aborting the individual webhook call.
+Braze will retry webhook sends for up to 24 hours before aborting the individual webhook call. When retrying after any 500-level error, Braze will make 5 attempts using exponential backoff for a period of approximately 30 minutes before aborting the individual webhook call.
```
Comment (internetisaiah) [inline _docs/_user_guide/message_building_by_channel/webhooks/creating_a_webhook.md:203]:
```suggestion

{% alert note %}
For 500-level errors, Braze will retry the webhook send up to 5 times over 30 minutes using exponential backoff. For all other errors, Braze will continue to retry for up to 24 hours.
```

File: _docs/_user_guide/message_building_by_channel/webhooks/creating_a_webhook.md [hunk 3]
```diff
@@ -200,7 +200,7 @@ Like other campaigns, Braze tracks the delivery of your webhook campaigns and co
...
 
 Each webhook is allowed 90 seconds before it times out.
```
Comment (internetisaiah) [inline _docs/_user_guide/message_building_by_channel/webhooks/creating_a_webhook.md:205]:
```suggestion
```

File: _docs/_user_guide/message_building_by_channel/webhooks/creating_a_webhook.md [hunk 4]
```diff
@@ -200,9 +200,12 @@ Like other campaigns, Braze tracks the delivery of your webhook campaigns and co
 | 5xx (server error)   | No | Yes |
 {: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}
 
-Braze will retry webhook sends for up to 24 hours. When retrying, Braze will make five attempts using exponential backoff for a period of approximately 30 minutes before aborting the individual webhook call.
+{% alert note %}
+For 500-level errors, Braze will retry the webhook send up to 5 times over 30 minutes using exponential backoff. For all other errors, Braze will continue to retry for up to 24 hours.
```
Comment (internetisaiah) [inline _docs/_user_guide/message_building_by_channel/webhooks/creating_a_webhook.md:204]: Hello @SimeonBrazeSA , thanks for contributing! I slightly simplified your content fix. Are my changes still accurate? If not, let me know how and I will modify. Thanks!

Your original:

> Braze will retry webhook sends for up to 24 hours before aborting the individual webhook call. When retrying after 5xx errors, Braze will make five attempts using exponential backoff for a period of approximately 30 minutes before aborting the individual webhook call.

Comment (josh-mccrowell-braze) [reply _docs/_user_guide/message_building_by_channel/webhooks/creating_a_webhook.md:204]: Hi @SimeonBrazeSA - Just making sure you saw the clarifying question above. Thanks again for contributing!

**Review by josh-mccrowell-braze — APPROVED** [review]

**Pull request merged**